DryWetMidi.Tests/Interaction/Utilities/NodeId/NoteIdTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Melanchall.DryWetMidi.Common;
+using Melanchall.DryWetMidi.Interaction;
+using NUnit.Framework;
+
+namespace Melanchall.DryWetMidi.Tests.Interaction
+{
+    [TestFixture]
+    public sealed class NoteIdTests
+    {
+        #region Test methods
+
+        #region Equals
+
+        [Test]
+        public void Equals_NullObject()
+        {
+            var noteId = new NoteId(new FourBitNumber(), new SevenBitNumber());
+            var isEqual = noteId.Equals(null);
+            Assert.IsFalse(isEqual);
```

Comment:
You can use `Assert.AreEqual` to check equality of objects. `AreEqual` uses `Equals` method so it's the same.

Reply:
And in other places too.

Reply:
Also please specify error message for every assert you use.